src/main/scala/io/iohk/ethereum/keystore/Wallet.scala

```diff
@@ -2,8 +2,13 @@ package io.iohk.ethereum.keystore
 
 
 import akka.util.ByteString
+import io.iohk.ethereum.crypto._
 import io.iohk.ethereum.domain.{Address, SignedTransaction, Transaction}
+import org.spongycastle.crypto.AsymmetricCipherKeyPair
 
-case class Wallet(address: Address, privKey: ByteString) {
-  def signTx(tx: Transaction): SignedTransaction = ???
+case class Wallet(address: Address, prvKey: ByteString) {
+  lazy val keyPair: AsymmetricCipherKeyPair = keyPairFromPrvKey(prvKey.toArray)
+
+  def signTx(tx: Transaction): SignedTransaction =
+    SignedTransaction.sign(tx, keyPair, None)
```

Comment:
yes, exactly that, parity accepts new signing schema only when chainId = networkId
but in case of ETC chainId = 3d and networkId = 1